### PR TITLE
[4.0] Permissions widths

### PR DESCRIPTION
**Pull request by brianteeman**
There is no specific width set on the columns in the permissions table. As a result as you move between access levels you will see the columns jump around depending on the width of the content on the first row in the last column.

This PR resolves this by setting explicit widths for the columns

### recording of the problem that this pr fixes
sidth

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5406f7a12d065b7f2b03db69078d15b848efa187<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26027">issues.joomla.org/tracker/joomla-cms/26027</a>.</sub>

**Comment by C-Lodder** [comment]
The `width` attribute on `<td>`'s has been deprecated since HTML4 and obsolete in HTML5.

Replace with `style="width: xxx"`

**Comment by brianteeman** [comment]
Thanks @C-Lodder

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 93f416b423eea1781c94c255abea77a34951c771<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26027">issues.joomla.org/tracker/joomla-cms/26027</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 93f416b423eea1781c94c255abea77a34951c771<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26027">issues.joomla.org/tracker/joomla-cms/26027</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 1]
```diff
@@ -99,7 +99,6 @@ COM_CONTENT_FILTER_SEARCH_LABEL="Search Articles"
 COM_CONTENT_FLOAT_FULLTEXT_LABEL="Full Text Image Float"
 COM_CONTENT_FLOAT_INTRO_LABEL="Intro Image Float"
 COM_CONTENT_FLOAT_LABEL="Image Float"
-COM_CONTENT_GLOBAL_OPTIONS="Global"
```
Comment (HLeithner) [inline administrator/language/en-GB/en-GB.com_content.ini]: What's the reason to remove this string in the PR?

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_content.ini]: oops wrong  branch

**Comment by HLeithner** [comment]
thanks

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**